burger/toppings/version.py

```diff
@@ -174,7 +174,38 @@ def get_protocol_version(aggregate, classloader, verbose):
                                     str[len("Outdated server! I'm still on "):]
                                 versions["id"] = versions["name"]
                                 return
-        elif verbose:
+        elif versions["distribution"] == "client" and "nethandler.client" in aggregate["classes"]:
+            # If we know this is the client, and there's no nethandler.server, this is a version prior to the codebase merge (12w17a or prior)
+            # We need to look for the protocol name and version elsewhere
+
+            # We can get the name from the startup class
+            for constant in classloader.search_constant_pool(path="net/minecraft/client/Minecraft", type_=String):
+                if "Minecraft Minecraft " in constant.string.value:
+                    versions["id"] = versions["name"] = constant.string.value[len("Minecraft Minecraft "):]
+                    break
```

Comment:
What string is this? I don't remember seeing the duplicated "Minecraft Minecraft" anywhere, but it seems like it exists even in 1.5 (but not in 1.6).

Reply:
It's found like this: `Display.setTitle("Minecraft Minecraft 12w21a");`
Not sure what for though, since I don't see that reflected on the window's title.
It shouldn't matter either way, since it's only used on the logic up until 12w17a. After that, it falls back to the current search on server nethandler (which is broken for that version range you mentioned).